src/main/java/se/sundsvall/notes/service/mapper/OffsetDateTimeSerializer.java

```diff
@@ -0,0 +1,22 @@
+package se.sundsvall.notes.service.mapper;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonPrimitive;
+import com.google.gson.JsonSerializationContext;
+import com.google.gson.JsonSerializer;
+
+import java.lang.reflect.Type;
+import java.time.OffsetDateTime;
+
+import static java.time.format.DateTimeFormatter.ISO_DATE_TIME;
+
```

Comment:
Why do you need this? Why not use jackson (ObjectMapper) instead of Gson for the serialization-logic?
Date-serializers/deserializers are offered in dept44 for jackson.

Reply:
Fixed